Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -63,9 +66,12 @@ def default(context, obj):
             return super(Tracker.Encoder, context).default(obj)
 
 
-    def __init__(self, url, users=None, res=None):
+    def __init__(self, url, users=None, res=None, component_color=None, version_color=None):
```

Comment:
Do we plan to change `DEFAULT_COMPONENT_COLOR` and `DEFAULT_VERSION_COLOR` in derived classes?
If not, we may just do `component_color= DEFAULT_COMPONENT_COLOR, version_color= DEFAULT_VERSION_COLOR`